test/mocker/rtm.spec.js

```diff
@@ -33,10 +33,13 @@ describe('mocker: rtm', function () {
       Server: sinon.stub()
     }
 
-    serverMock = 'server'
+    serverMock = {
+      close: sinon.stub().returns()
+    }
 
     expressAppMock = {
-      listen: sinon.stub().returns(serverMock)
+      listen: sinon.stub().returns(serverMock),
+      close: serverMock.close()
```

Comment:
You shouldn't need this mock, close is only called on the server returned by `listen`, not on the expressApp.